### PR TITLE
Catch missing training datasets

### PR DESCRIPTION
**Pull request by neubig**
Datalab import crashed on datasets that only have a test set. This PR fixes this and throws a warning.

### CODE REVIEW

**Review by OscarWang114 — APPROVED** [review]
LGTM, and it seems uncontroversial!
